osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -288,7 +288,7 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
         public override void PlaySamples()
         {
             // rather than doing it this way, we should probably attach the sample to the tail circle.
-            // this can only be done after we stop using LegacyLastTick.
+            // this can only be done if we stop using LastTick.
```

Comment:
At this point we're probably never stopping using `LastTick` so comment should probably be reworded?

Something like

```diff
diff --git a/osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs b/osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs
index 77e60a1690..236d89a7c3 100644
--- a/osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs
+++ b/osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs
@@ -287,8 +287,9 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
 
         public override void PlaySamples()
         {
-            // rather than doing it this way, we should probably attach the sample to the tail circle.
-            // this can only be done if we stop using LastTick.
+            // this is only done like this (instead of just attaching the sample to the tail)
+            // due to the existence of LastTick and the fact of it having a -36ms offset
+            // see `SliderEventGenerator.LAST_TICK_OFFSET`
             if (!TailCircle.SamplePlaysOnlyOnHit || TailCircle.IsHit)
                 base.PlaySamples();
         }

```

perhaps?

Reply:
Actually with the follow-up PR, it may be possible. Would require some reconsideration of the judgement logic, but it's definitely within reach (would have to record the "tracked" state and only actually apply at the correct endtime, to ensure the samples don't play early).

I think I'd want to leave this...

Reply:
Fair enough.